app/helpers/advice_page_helper.rb

```diff
@@ -143,6 +143,14 @@ def warm_weather_on_days_status(rating)
     end
   end
 
+  def advice_pages_for_school_and_fuel(advice_pages, school, fuel_type)
+    if school.multiple_meters?(fuel_type) && Flipper.enabled?(:meter_breakdowns, current_user)
```

Comment:
No, that only returns pages that are for multiple meters. The check is returning all pages (so where multiple_meters is true or false on the advice page model. Otherwise its returning only those that are single meters.